<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="TreeMenu" name="treeMenu">
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1">
        <item>
         <widget class="QTabWidget" name="tabWidget_1">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>编译输出</string>
           </attribute>
           <widget class="QTextEdit" name="textEditOutput">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>1021</width>
              <height>100</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>400</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>调试</string>
           </attribute>
           <widget class="QTextEdit" name="textEditOutput_2">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>0</y>
              <width>1021</width>
              <height>100</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>400</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveFileAs"/>
    <addaction name="actionQuit"/>
    <addaction name="actionSaveFile"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="actionReverse"/>
   </widget>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>搜索(S)</string>
    </property>
    <addaction name="actionsearch"/>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
   </widget>
   <widget class="QMenu" name="menu_R">
    <property name="title">
     <string>运行(R)</string>
    </property>
    <addaction name="action_edit"/>
    <addaction name="action_run"/>
    <addaction name="action_edit_run"/>
    <addaction name="action_debug"/>
    <addaction name="action_addbreakpoint"/>
    <addaction name="action_delete"/>
    <addaction name="action_check"/>
   </widget>
   <widget class="QMenu" name="menu_T">
    <property name="title">
     <string>工具(T)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(H)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>更换主题</string>
    </property>
    <addaction name="actionChangeWhite"/>
    <addaction name="actionChangePink"/>
    <addaction name="actionChangeOrange"/>
    <addaction name="actionChangeColorful"/>
    <addaction name="actionChangeDarkBlue"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menu_E"/>
   <addaction name="menu_S"/>
   <addaction name="menu_R"/>
   <addaction name="menu_T"/>
   <addaction name="menu_H"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_tool_new"/>
   <addaction name="separator"/>
   <addaction name="action_tool_save"/>
   <addaction name="separator"/>
   <addaction name="action_tool_copy"/>
   <addaction name="separator"/>
   <addaction name="action_tool_cut"/>
   <addaction name="separator"/>
   <addaction name="action_tool_paste"/>
   <addaction name="separator"/>
   <addaction name="action_tool_open"/>
   <addaction name="separator"/>
   <addaction name="action_tool_close"/>
   <addaction name="separator"/>
   <addaction name="action_tool_run"/>
   <addaction name="separator"/>
   <addaction name="action_tool_Debug"/>
   <addaction name="separator"/>
   <addaction name="actiontext_view"/>
   <addaction name="separator"/>
   <addaction name="actionedit_view"/>
   <addaction name="separator"/>
   <addaction name="action_tool_singleRun"/>
   <addaction name="separator"/>
   <addaction name="action_tool_processRun"/>
   <addaction name="separator"/>
   <addaction name="action_tool_RunWhileDebug"/>
   <addaction name="separator"/>
   <addaction name="action_tool_search"/>
   <addaction name="separator"/>
   <addaction name="action_tool_Exit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewFile">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="action_tool_new">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/newFile.png</normaloff>:/img/newFile.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="action_tool_save">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/saveFile.png</normaloff>:/img/saveFile.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionsearch">
   <property name="text">
    <string>全局查找</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionSaveFileAs">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="actionReverse">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>顺序查找</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>替换</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>全局替换</string>
   </property>
  </action>
  <action name="action_edit">
   <property name="text">
    <string>编译</string>
   </property>
  </action>
  <action name="action_run">
   <property name="text">
    <string>运行</string>
   </property>
  </action>
  <action name="action_edit_run">
   <property name="text">
    <string>编译运行</string>
   </property>
  </action>
  <action name="action_debug">
   <property name="text">
    <string>debug</string>
   </property>
  </action>
  <action name="action_tool_copy">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/copy.png</normaloff>:/img/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="action_tool_cut">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/cut.png</normaloff>:/img/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="action_tool_open">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/openFile.png</normaloff>:/img/openFile.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_tool_close">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/closeFile.png</normaloff>:/img/closeFile.png</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="action_tool_run">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/Run.png</normaloff>:/img/Run.png</iconset>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actiontext_view">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/textView.png</normaloff>:/img/textView.png</iconset>
   </property>
   <property name="text">
    <string>textView</string>
   </property>
  </action>
  <action name="actionedit_view">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/editView.png</normaloff>:/img/editView.png</iconset>
   </property>
   <property name="text">
    <string>edit</string>
   </property>
  </action>
  <action name="actionChangeWhite">
   <property name="text">
    <string>冰雪奇缘</string>
   </property>
  </action>
  <action name="actionChangePink">
   <property name="text">
    <string>玲娜贝儿</string>
   </property>
  </action>
  <action name="actionChangeOrange">
   <property name="text">
    <string>小熊维尼</string>
   </property>
  </action>
  <action name="actionChangeColorful">
   <property name="text">
    <string>彩色庆典</string>
   </property>
  </action>
  <action name="actionChangeDarkBlue">
   <property name="text">
    <string>史迪仔</string>
   </property>
  </action>
  <action name="action_tool_paste">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/paste.png</normaloff>:/img/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="action_tool_Debug">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/Debug.png</normaloff>:/img/Debug.png</iconset>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="action_addbreakpoint">
   <property name="text">
    <string>增加断点</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="text">
    <string>删除断点</string>
   </property>
  </action>
  <action name="action_check">
   <property name="text">
    <string>查看断点信息</string>
   </property>
  </action>
  <action name="action_tool_singleRun">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/singleRun.png</normaloff>:/img/singleRun.png</iconset>
   </property>
   <property name="text">
    <string>逐步运行</string>
   </property>
  </action>
  <action name="action_tool_processRun">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/processRun.png</normaloff>:/img/processRun.png</iconset>
   </property>
   <property name="text">
    <string>逐过程运行</string>
   </property>
  </action>
  <action name="action_tool_RunWhileDebug">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/debugRun.png</normaloff>:/img/debugRun.png</iconset>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </action>
  <action name="action_tool_search">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/checkVariable.png</normaloff>:/img/checkVariable.png</iconset>
   </property>
   <property name="text">
    <string>查看变量</string>
   </property>
  </action>
  <action name="action_tool_Exit">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/exit.png</normaloff>:/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TreeMenu</class>
   <extends>QTreeWidget</extends>
   <header>treemenu.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
